earthly-entrypoint.sh

```diff
@@ -46,7 +46,24 @@ if [ -z "$NO_BUILDKIT" ]; then
         ) || true
     fi
 
+    # generate certificates
+    earthly --config "$earthly_config" --buildkit-host=tcp://127.0.0.1:8372 bootstrap --certs-hostname=$(hostname)
+
+    if [ ! -f /etc/ca.pem ]; then
+      ln -s /root/.earthly/certs/ca_cert.pem /etc/ca.pem
+    fi
+
+    if [ ! -f /etc/cert.pem ]; then
+      ln -s /root/.earthly/certs/buildkit_cert.pem /etc/cert.pem
+    fi
+
+    if [ ! -f /etc/key.pem ]; then
+      ln -s /root/.earthly/certs/buildkit_key.pem /etc/key.pem
+    fi
+
+
```

Comment:
Why are we using TLS in this container with the buildkit thats also running inside the container?

Reply:
I added this because if tls is disabled, then connection to a satellite will fail or will require to enable it by passing 
`-e GLOBAL_CONFIG="{tls_enabled: true}"`.

I _think_ this is more seamless but I acknowledge it's somewhat redundant to use tls internally.

What are your thoughts?